### PR TITLE
feat: render components instead of elements

### PR DESCRIPTION
**Pull request by philippfromme**
---

Requires https://github.com/bpmn-io/properties-panel/pull/134

### CODE REVIEW

File: src/provider/camunda-platform/CamundaPlatformPropertiesProvider.js [hunk 1]
```diff
@@ -123,6 +123,14 @@ export default class CamundaPlatformPropertiesProvider {
...
       moveImplementationGroup(groups);
 
+      // console.log(JSON.stringify(groups, (key, value) => {
+      //   if (key === 'component' && typeof value === 'function') {
+      //     return `function ${ value.name }(props) { ... }`;
+      //   }
+
+      //   return value;
+      // }, 2));
+
```
Comment (barmac) [inline src/provider/camunda-platform/CamundaPlatformPropertiesProvider.js:133]:
```suggestion
```

File: src/provider/camunda-platform/properties/ListenerProps.js [hunk 2]
```diff
@@ -390,6 +400,31 @@ function EventTypeDetails(props) {
   return [];
 }
 
+function Field(props) {
+  const {
+    element,
+    id: idPrefix,
+    index,
+    item: field,
+    isNew
```
Comment (barmac) [inline src/provider/camunda-platform/properties/ListenerProps.js:409]:
```suggestion
    open
```

File: src/provider/camunda-platform/properties/ListenerProps.js [hunk 3]
```diff
@@ -390,6 +400,31 @@ function EventTypeDetails(props) {
...
+  return (
+    <CollapsibleEntry
+      id={ fieldId }
+      entries={ FieldInjection({
+        element,
+        field,
+        idPrefix: fieldId
+      }) }
+      label={ field.get('name') || '<empty>' }
+      open={ !!isNew }
```
Comment (barmac) [inline src/provider/camunda-platform/properties/ListenerProps.js:423]:
```suggestion
      open={ open }
```

File: src/provider/camunda-platform/properties/ListenerProps.js (hunk 2 above)
Comment (barmac) [reply src/provider/camunda-platform/properties/ListenerProps.js:409]: We don't pass `isNew`, do we?

**Review by barmac — CHANGES_REQUESTED** [review]
I left a couple of comments but it looks good generally.

File: src/provider/camunda-platform/CamundaPlatformPropertiesProvider.js (hunk 1 above)
Comment (philippfromme) [reply src/provider/camunda-platform/CamundaPlatformPropertiesProvider.js:133]: Ooops! 🤡

File: src/provider/camunda-platform/properties/ListenerProps.js (hunk 2 above)
Comment (philippfromme) [reply src/provider/camunda-platform/properties/ListenerProps.js:409]: Wow, good catch! 🏅

**Review by barmac — APPROVED** [review]

**Pull request merged**